lib/core/test_runner.js

```diff
@@ -15,7 +15,7 @@ export class TestRunner {
   #failFastMode;
   #randomOrder;
   #testExecutionTimeoutMs;
-
+  #currentExecutedFile;
```

Comment:
I prefer the word "loaded" because executed can be interpreted as test execution, that happens later on.